pages/HowToPlay.jsx

```diff
@@ -16,15 +16,11 @@ class HowToPlay extends React.Component {
                         <ul className='htp-main-list'>
                             <li><a href='#decks'>Adding Decks</a></li>
                             <li><a href='#profile'>Profile Options</a></li>
-                            <ul className='htp-sub-list'>
-                                <li><a href='#action'>Action Windows</a></li>
-                                <li><a href='#timed'>Timed Interrupt Window</a></li>
-                            </ul>
                             <li><a href='#bugs'>Bugs and Automation</a></li>
                             <li><a href='#interactions'>Specific Card Interactions</a></li>
                             <ul className='htp-sub-list'>
-                                <li><a href='#riddle'>Varys' Riddle vs Summer Harvest</a></li>
-                                <li><a href='#mag'>Mag the Mighty vs. saves</a></li>
+                                <li><a href='#marges'>Old Marge's Manor</a></li>
+                                <li><a href='#johnnybrocklehurst'>Johnny Brocklehurst</a></li>
```

Comment:
Let's make this as one list item.
```suggestion
                                <li><a href='#margesandjohnny'>Old Marge's Manor and Johnny Brocklehurst</a></li>
```